apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java

```diff
@@ -582,11 +582,9 @@ private String partDFieldByMonth(CcwCodebookVariable month) {
 
     if (requestHeader.isMBIinIncludeIdentifiers()) {
       joinsClause += "left join fetch b.medicareBeneficiaryIdHistories ";
-      passDistinctThrough = true;
```

Comment:
Is the removal of these intended?

Reply:
yes; distinctPassThru true cause the db engine to perform the distinct (unique) processing while setting to false causes the JDBC driver to perform the distinct. For a simple parent-child relationship (which this is) it is more performance to have the JDBC driver handle things.